assets/css/main.css

```diff
@@ -1,3 +1,12 @@
 * {
     font-family: 'Poppins', sans-serif;
+}
+
+#main-container {
+    margin-top: 500px;
+}
+
+#startButton{
+    margin-top: 80px;
+    border: 5px solid white;
```

Comment:
I think we can get rid of line 5-11 and use tailwind CSS to style instead. This .css file should only be used to overwrite tailwind styling.

Reply:
Thanks for making the changes requested!